### DISCUSSION TITLE
Created wall height turns back 0 value

### DISCUSSION THREAD

**Discussion by AlexKDo** [UMEP-dev/UMEP#505] [Q&A] [answered]
I am trying to create wall aspect and height from DSM but i have wall aspect and wall height from the same dsm returns with 0 values.
![image](https://user-images.githubusercontent.com/81244161/224582404-9b274a1b-b981-4aaa-a8f5-78bd7975b9c8.png)

![image](https://user-images.githubusercontent.com/81244161/224582452-2df6d58f-8081-4579-b4d6-914f95dae86d.png)

**Comment by biglimp** [comment, marked as answer]
This is probably because you have NoData pixel somewhere in your data. If you scale the images you will see the walls.
